Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -551,22 +551,33 @@ export type Props = $ReadOnly<{|
  */
 const ScrollView = createReactClass({
   displayName: 'ScrollView',
-  mixins: [ScrollResponder.Mixin],
-
+  _scrollResponder: {},
```

Comment:
It might be a good idea to just turn this into a React.Component subclass right now: 

```JavaScript
const ScrollResponder = require('ScrollResponder');

function createScrollResponder(node: React.ElementRef<typeof ScrollView>): typeof ScrollResponder.Mixin {
  const scrollResponder = {... ScrollResponder.Mixin};
  for (const key of Object.keys(ScrollResponder.Mixin)) {
    if (typeof scrollResponder[key] === 'function') {
      scrollResponder[key] = scrollResponder[key].bind(node);
    }
  }
  return scrollResponder;
}

class ScrollView extends React.Component<Props, State> {
  _scrollResponder: typeof ScrollResponder.Mixin = createScrollResponder(this);
}
```

Then, we change all references to scrollResponder methods within `ScrollView` to use `this._scrollResponder`